pkg/models/suite_test.go

```diff
@@ -0,0 +1,127 @@
+package models
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/content-services/content-sources-backend/pkg/config"
+	"github.com/lib/pq"
+	"github.com/openlyinc/pointy"
+	"github.com/stretchr/testify/suite"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+type ModelsSuite struct {
+	suite.Suite
+	db *gorm.DB
+	tx *gorm.DB
+}
+
+func getDSNWithOptions(user string, password string, dbname string, host string, port int) string {
+	return fmt.Sprintf(
```

Comment:
It seems like these database connection functions (here and in seeds) accomplish almost the same result as the functions in the db package; the only database being connected to is the one returned by config.Get(). Is that right?

Instead of creating new functions here, perhaps the db package can be used and/or modified slightly? Maybe you could add `GetDbConnection(skipDefaultTransaction bool)`? I think that would cover the use cases here in the tests.

Reply:
Thanks! duplication removed! I realized I didn't need the options finally just keeping the connector that represent the transaction.